<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstClasse</class>
 <widget class="QMainWindow" name="FirstClasse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FirstClasse</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(0, 85, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 250, 254);</string>
   </property>
   <widget class="QWidget" name="icon_only_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>51</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	background-color: rgb(31, 149, 239);
}

QPushButton {
text-align:center;
color:white;
height:30px;

}

QPushButton:checked {
background-color:#F5FAFE;
color:#1F95EF;
font-weight:bold;
}</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>41</width>
       <height>40</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ressource.qrc">:/images/profil.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="sign2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>380</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressource.qrc">
       <normaloff>:/images/log_out.png</normaloff>:/images/log_out.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>51</width>
       <height>242</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QPushButton" name="dashboard1">
        <property name="icon">
         <iconset resource="ressource.qrc">
          <normaloff>:/images/dashboard.png</normaloff>:/images/dashboard.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="profil1">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressource.qrc">
          <normaloff>:/images/profile.png</normaloff>:/images/profile.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cours1">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressource.qrc">
          <normaloff>:/images/task-query-1.png</normaloff>:/images/task-query-1.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="syllabus1">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressource.qrc">
          <normaloff>:/images/syllabus.png</normaloff>:/images/syllabus.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="avis1">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressource.qrc">
          <normaloff>:/images/avis.png</normaloff>:/images/avis.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="icon_name_widget" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>151</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	background-color: rgb(31, 149, 239);
}

QPushButton {
text-align:left;
color:white;
height:30px;
padding-left:10px;

}

QPushButton:checked {
background-color:#F5FAFE;
color:#1F95EF;
font-weight:bold;
}

</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="ressource.qrc">:/images/image.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Responsable Classe</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <widget class="QPushButton" name="dashboard2">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Dashboard</string>
         </property>
         <property name="icon">
          <iconset resource="ressource.qrc">
           <normaloff>:/images/dashboard.png</normaloff>:/images/dashboard.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="profil2">
         <property name="text">
          <string>Profil</string>
         </property>
         <property name="icon">
          <iconset resource="ressource.qrc">
           <normaloff>:/images/profile.png</normaloff>:/images/profile.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cours2">
         <property name="text">
          <string>Constater Effectivite Cours</string>
         </property>
         <property name="icon">
          <iconset resource="ressource.qrc">
           <normaloff>:/images/task-query-1.png</normaloff>:/images/task-query-1.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="syllabus2">
         <property name="text">
          <string>Syllabus</string>
         </property>
         <property name="icon">
          <iconset resource="ressource.qrc">
           <normaloff>:/images/syllabus.png</normaloff>:/images/syllabus.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="avis2">
         <property name="text">
          <string>Avis Etudiant</string>
         </property>
         <property name="icon">
          <iconset resource="ressource.qrc">
           <normaloff>:/images/avis.png</normaloff>:/images/avis.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>188</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sign1">
       <property name="styleSheet">
        <string notr="true">QWidget {
	rgb(0, 0, 0)
}</string>
       </property>
       <property name="text">
        <string>Deconnexion</string>
       </property>
       <property name="icon">
        <iconset resource="ressource.qrc">
         <normaloff>:/images/log_out.png</normaloff>:/images/log_out.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="header_widget" native="true">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>10</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(208, 242, 255);</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_13">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(94, 150, 255);
border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ressource.qrc">
         <normaloff>:/images/menu.png</normaloff>:/images/menu.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_14">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ressource.qrc">
         <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>85</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_15">
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ressource.qrc">
         <normaloff>:/images/profil.png</normaloff>:/images/profil.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="mainWidjet" native="true">
    <property name="geometry">
     <rect>
      <x>217</x>
      <y>60</y>
      <width>371</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(201, 246, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
  <include location="ressource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_13</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_only_widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_13</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_name_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>avis1</sender>
   <signal>toggled(bool)</signal>
   <receiver>avis2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>syllabus1</sender>
   <signal>toggled(bool)</signal>
   <receiver>syllabus2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cours1</sender>
   <signal>toggled(bool)</signal>
   <receiver>cours2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>profil1</sender>
   <signal>toggled(bool)</signal>
   <receiver>profil2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dashboard1</sender>
   <signal>toggled(bool)</signal>
   <receiver>dashboard2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dashboard2</sender>
   <signal>toggled(bool)</signal>
   <receiver>dashboard1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>profil2</sender>
   <signal>toggled(bool)</signal>
   <receiver>profil1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cours2</sender>
   <signal>toggled(bool)</signal>
   <receiver>cours1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>syllabus2</sender>
   <signal>toggled(bool)</signal>
   <receiver>syllabus1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>avis2</sender>
   <signal>toggled(bool)</signal>
   <receiver>avis1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sign1</sender>
   <signal>toggled(bool)</signal>
   <receiver>FirstClasse</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sign2</sender>
   <signal>toggled(bool)</signal>
   <receiver>FirstClasse</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
